appdb.py

```diff
@@ -15,19 +15,32 @@
 mysql=MySQL(app)
 
 @app.route('/', methods=['GET','POST'])
+def home():
+    if request.method=='GET':
+        return render_template('home.html')
+    if request.method=='POST':
+        return render_template('home.html')
+
+@app.route('/newuser')
+def success():
+    return render_template('success.html')
+
+
+@app.route('/login', methods=['GET','POST'])
 def index():
     if request.method=='POST':
         #Fetch form data
         userDetails = request.form
-        name=userDetails['name']
-        email=userDetails['email']
+        username=userDetails['username']
+        password=userDetails['password']

```

Comment:
I was going to mention this in the next task because this task only includes having a home page :) But passwords should be hashed + salted. Take a look at [this article](https://itnext.io/how-not-to-store-passwords-4955569e6e84) for example. This is in case there's a breach (a leak of the database by a hacker or technical reason) and also even the developers/administrators of the app itself should never be able to read the passwords.

Using hashing we can do the one thing we want to do: store passwords in an unreadable format while still being able to check if the text entered to log in is the one in the database.

If you have any questions let me know!